tests/subsys/logging/log_backend_uart/multi.overlay

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023 Meta
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,log-uart = &log_uarts;
```

Comment:
I think my preference would be either to use phandles property in the chosen node, or to simply use a child DT node under each UART selected to be a log backend.

However, the phandles approach does not seem to work for in the chosen node.

Personally, I think the chosen + node + phandles is kind of overkill.

Reply:
Alternatively, if it were possible to support a phandles array in chosen, that would be kind of ideal, but it might require some additional devicetree hacks.